File: haskell/1005/main.hs
import Text.Printf (printf)

averageOne :: IO String
averageOne = do
    a <- readLn :: IO Double
    b <- readLn :: IO Double

    let weightOfA = 3.5
        weightOfB = 7.5

    let media = (a * weightOfA + b * weightOfB) / (weightOfA + weightOfB)

    return $ printf "MEDIA = %.5f" media

main :: IO ()
main = do
    result <- averageOne
    putStrLn result

File: c++/1005/main.cpp
#include <iostream>
#include <iomanip>

using namespace std;

double averageOne()
{
    double a, b;
    double weightOfA = 3.5, weightOfB = 7.5;

    cin >> a >> b;

    double media = (a * weightOfA + b * weightOfB) / (weightOfA + weightOfB);

    return media;
}

int main()
{
    double result = averageOne();

    cout << fixed << setprecision(5);
    cout << "MEDIA = " << result << endl;

    return 0;
}
